### ISSUE TITLE
Cheetah-Dead end content

### ISSUE THREAD

**Issue by laurenernest** [department-of-veterans-affairs/va.gov-team#20789]
We have a few different variations of dead ends after the facility page.

1. No clinics available for the facility you chose:
![Screen Shot 2021-03-04 at 9 06 20 AM](https://user-images.githubusercontent.com/634932/109976074-5cac5280-7cc9-11eb-9eca-15d263448ad1.png)
2. Only one facility with online appointments, but no clinics available:
![Screen Shot 2021-03-04 at 9 09 02 AM](https://user-images.githubusercontent.com/634932/109976069-5c13bc00-7cc9-11eb-808b-938bea460f53.png)
3. No slots available for chosen clinic (just an empty calendar)
![Screen Shot 2021-03-04 at 9 08 11 AM](https://user-images.githubusercontent.com/634932/109976073-5cac5280-7cc9-11eb-8de6-7428dcf6d0cc.png)

Scenario 3 could be tricky, because we fetch two months in advance, so there could always be slots later.

**Comment by jbalboni** [comment]
@laurenernest @outerpress I updated the description with the 3 scenarios I know of.

**Comment by outerpress** [comment]
My notes and memory from last week are a little sketchy, so could y'all confirm/deny a few things about these states?:

- Scenario 1: We can't query multiple facilities for open slots at once, so we can't surface this issue until the user selects a clinic and clicks continue (?)

- Scenario 2: We can check this, and catch them on the home page before they start scheduling (?)

- Scenario 3: Same as scenario 1 (?)

So basically we discussed leaving 1 and 3 as is for Cheetah, and catching users on the home page in the case of 2. 

@jbalboni @laurenernest

**Comment by jbalboni** [comment]
@outerpress Yes, that sounds right to me.

**Comment by outerpress** [comment]
@ldelacosta @JRRoof this was done a long time ago as part of the vaccine flow (called "cheetah" at the time). I'm closing it.
